studies/models.py

```diff
@@ -26,7 +26,6 @@
 
 from accounts.models import Child, DemographicData, User
 from attachment_helpers import get_download_url
-from project import settings
```

Comment:
Was this line removed automatically by the linter or by you @okaycj?
This file needs access to `settings`, so I'm guessing this was removed because settings is imported earlier:
```python
from django.conf import settings
```
Do we know which of these imports is correct?

Reply:
They're probably both correct, but `django.conf` is correct per docs.  https://docs.djangoproject.com/en/5.0/topics/settings/#using-settings-in-python-code.